CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
-## [UNRELEASED] neptune-tensorboard 0.5.3
+## [UNRELEASED] neptune-tensorboard 0.6.0
```

Comment:
This is a non-backward compatible version (this PR) that changes a lot. I recommend first:
- Update the CHANGELOG on master to remove `[UNRELEASED]`
- Release the `0.5.3` version
- Switch to `1.0.0` version with this PR

Reply:
Actually, what you said makes sense. Will do that. Thanks!

Reply:
Ok so 2 out of 3 things are done and you're free to go with rebase and `1.0.0` version. Btw. What do you think about releasing this particular PR under `1.0.0.pre0` or something to be able to write examples? Support of Pytorch-based Tensorboard would be released in `1.0.0.pre1` as an example.

Reply:
Yeah, that makes sense.